PoroFracture.C

```diff
@@ -49,8 +49,9 @@ public:
   virtual bool getElementSolution(Vectors& eV,
                                   const std::vector<int>& MNPC) const
   {
-    if (!this->FractureElasticityVoigt::getElementSolution(eV,MNPC))
-      return false;
```

Comment:
you need to extract from primsol to eV here. the whole filtering below will be wrong.
i suggest
```
utl::gather(MNPC, nsd+1, primsol.front() eV.front());
```
then you can do the filtering below.